src/session/session_compact.c

```diff
@@ -404,6 +404,7 @@ __wt_session_compact(
 	WT_ERR(__wt_config_gets(session, cfg, "timeout", &cval));
 	session->compact->max_time = (uint64_t)cval.val;
 	__wt_epoch(session, &session->compact->begin);
+	session->compact->prog_msg_count = 0;
```

Comment:
This isn't necessary - the structure is 0 initialized - so explicit initialization is redundant.